src/release.ts

```diff
@@ -0,0 +1,82 @@
+import * as github from "@actions/github";
+import * as semver from "semver";
```

Comment:
I will have a look at `ncc`, but I don't think it will be necessary, at least to get it to work, since all release branches contain `node_modules` for now. Not ideal, but it seems to be the most straightforward option without introducing additional tooling.

I am not, however, too keen on `node_modules` and transpiled files being checked in..

Reply:
Ah, I didn't notice you were checking in node_modules too. That's certaintly a solution. I like `ncc` method just because it produces a smaller download.